### PR TITLE
added loading state in Statistics Widget

### LINKED ISSUE #2246: Loading state for statistics widget
Similar to the widgets like buttons and charts, statistics widget should also have a loading state depending on which we show a spinner. 

Relevant file: `frontend/src/Editor/Components/Statistics.jsx`

### PR DESCRIPTION
**Pull request by manishkushare**
Resolve #2246

### CODE REVIEW

File: frontend/src/Editor/Components/Statistics.jsx [hunk 1]
```diff
@@ -64,45 +72,55 @@ export const Statistics = function Statistics({ height, properties, styles, dark
...
-          ...marginStyle,
-          color: primaryLabelColour !== '#8092AB' ? primaryLabelColour : darkMode && '#FFFFFC',
-        }}
-      >
-        {primaryValueLabel}
-      </p>
-      <h2 style={primaryStyle}>{primaryValue}</h2>
-      {hideSecondary ? (
-        ''
+      {loadingState === true ? (
```
Comment (arpitnath) [inline frontend/src/Editor/Components/Statistics.jsx:75]: Can we just destructure `loadingState` from properties here, and not use hooks?

Comment (manishkushare) [reply frontend/src/Editor/Components/Statistics.jsx:75]: Okay, working on it.

Comment (manishkushare) [reply frontend/src/Editor/Components/Statistics.jsx:74]: @arpitnath made the required changes

**Review by arpitnath — APPROVED** [review]

**Comment by arpitnath** [comment]
app: https://statistic-widget-loading-state.herokuapp.com/

**Pull request merged**